types/jsonld/jsonld-tests.ts

```diff
@@ -142,14 +147,6 @@ jsonld.normalize(doc, {expansion: false})
 /**
  * fromRDF() test
  */
-jsonld.fromRDF(docNQuads, (err, res) => {
-    log(res);
-});
-
-jsonld.fromRDF(docNQuads, {format: 'application/n-quads'}, (err, res) => {
-    log(res);
-});
```

Comment:
Doesn't seem to take a string: https://github.com/digitalbazaar/jsonld.js/blob/1333695a2c954dd2e7ce446396dcb8d00ad6f4ab/lib/fromRdf.js#L44-L49 (which makes the spec).